watcher/exts/playerstats.py

```diff
@@ -186,13 +187,61 @@ async def _equivalent_exchange(self, ctx, *, info):
         info_split = info.split(",")
         raw_rating = info_split[0]
         player_name = info_split[1].strip()
```

Comment:
this is on me honestly, but we should add a check for array length before grabbing specific indexes from it. currently this throws errors straight to the log and doesn't tell the user what went wrong